docs/content/config/best-practices/dkim_dmarc_spf.md

```diff
@@ -0,0 +1,336 @@
+    ```
+
+    If you have multiple domains and you want to sign with the modern ED25519 elliptic curve but also with RSA (you will likely want to have RSA as a fallback!):
+
+    ```cf
+    # documentation: https://rspamd.com/doc/modules/dkim_signing.html
+
+    enabled = true;
+
+    sign_authenticated = true;
+    sign_local = true;
+
+    use_domain = "header";
+    use_redis = false; # don't change unless Redis also provides the DKIM keys
+    use_esld = true;
+    check_pubkey = true;
+
+    domain {
+        example.com {
+            selectors [
+                {
+                    path = "/path/to/com.example.rsa.private";
+                    selector = "dkim-rsa";
+                },
+                {
+                  path = /path/to/com.example.ed25519.private";
+                  selector = "dkim-ed25519";
+                }
+          ]
+        }
+        example.org {
+            selectors [
+                {
+                    path = "/path/to/org.example.rsa.private";
+                    selector = "dkim-rsa";
+                },
+                {
+                  path = "/path/to/org.example.ed25519.private";
+                  selector = "dkim-ed25519";
+                }
+            ]
+        }
+    }
+    ```
+
+!!! bug "File Permissions & Signing Issues"
+
+    Make sure the user `_rspamd` is able to go into the directory where you persist the (private) key files, and ensure it can read them! You can use `su -l _rspamd -s /bin/bash` to change to the `_rspamd` user and then navigate into the directory and read the files (e.g. with `cat`). If any errors occur, you know the permissions are not correct yet.
+
+    When you set `check_pubkey = true;` in `dkim_signing.conf`, Rspamd will check the public key against your private key and emit a warning if there is a mismatch. Monitor the log file (`/var/log/supervisor/rspamd.log`) closely!
+
+[rspamd-docs-dkim-checks]: https://www.rspamd.com/doc/modules/dkim.html
+[rspamd-docs-dkim-signing]: https://www.rspamd.com/doc/modules/dkim_signing.html
+
+### DNS Record { #dkim-dns }
+
+When mail signed with your DKIM key is sent from your mail server, the receiver needs to check a DNS `TXT` record to verify the DKIM signature is trustworthy.
+
+!!! example "Configuring DNS - DKIM record"
+
+    When you generated your key in the previous step, the DNS data was saved into a file `<selector>.txt` (default: `mail.txt`). Use this content to update your [DNS via Web Interface][dns::example-webui] or directly editing your [DNS Zone file][dns::wikipedia-zonefile]:
+
+    === "Web Interface"
+
+        Create a new record:
+
+        - **Type:** `TXT`
+        - **Name:** should be your DKIM selector `<selector>._domainkey` (_default: `mail._domainkey`_)
+        - **Value:** should use the content within `( ... )` (_see the info section below for advice on correct formatting_)
+        - **TTL:** Use the default (_otherwise [3600 seconds is appropriate][dns::digicert-ttl]_)
```

Comment:
```suggestion
        | Field  | Value                                                                                                                                 |
        | -------- | --------------------------------------------------------------------------------------------------------------- |
        | Type   | `TXT`                                                                                                                                  |
        | Name | `<selector>._domainkey` (_default: `mail._domainkey`_)                                         |
        | TTL      | Use the default (_otherwise [3600 seconds is appropriate][dns::digicert-ttl]_) |
        | Data   | File content within `( ... )` (_formatted as advised below_)                                      |
```

Possibly better presentation than the list:

| Field  | Value                                                                                                                                 |
| -------- | --------------------------------------------------------------------------------------------------------------- |
| Type   | `TXT`                                                                                                                                  |
| Name | `<selector>._domainkey` (_default: `mail._domainkey`_)                                         |
| TTL      | Use the default (_otherwise [3600 seconds is appropriate][dns::digicert-ttl]_) |
| Data   | File content within `( ... )` (_formatted as advised below_)                                      |

[dns::digicert-ttl]: https://www.digicert.com/faq/dns/what-is-ttl